scripts/sepolia_deploy.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Abort the script on any error
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo $SCRIPT_DIR
+PROJECT_ROOT=$SCRIPT_DIR/..
+
+
+
+# Check for required commands
+command -v starkli >/dev/null 2>&1 || { echo >&2 "starkli not found. Aborting."; exit 1; }
+command -v scarb >/dev/null 2>&1 || { echo >&2 "scarb not found. Aborting."; exit 1; }
+
+# Configurable environment variables
+source $PROJECT_ROOT/.env
+: "${RPC_URL:=https://starknet-sepolia.public.blastapi.io/rpc/v0_7}"
+: "${ACCOUNT_PRIVATE_KEY:=ACCOUNT_PRIVATE_KEY is not set}"
+: "${ACCOUNT_ADDRESS:?ACCOUNT_ADDRESS is not set}"
+: "${TOKEN_NAME:?TOKEN_NAME is not set}"
+: "${TOKEN_SYMBOL:?TOKEN_SYMBOL is not set}"
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PROJECT_ROOT=$SCRIPT_DIR/..
+ONCHAIN_DIR=$PROJECT_ROOT/contracts
+KUDOS_SIERRA_FILE=$ONCHAIN_DIR/target/dev/kudos_Kudos.contract_class.json
+
+# Ensure tmp directory exists
+mkdir -p /tmp
```

Comment:
would prefer not to operate in a privileged dir
let's change to using the `$ONCHAIN_DIR/target` dir as we already rely on its existence and it will be hidden from git

Reply:
`mkdir -p $ONCHAIN_DIR/target/tmp`

Reply:
`mkdir -p $ONCHAIN_DIR/tar